proj.sh

```diff
@@ -41,7 +41,7 @@ function logs {
 
 function deploy-stack {
   generate-data
-  docker-compose -f docker-compose.yml -f docker-compose-prod.yml --context $1 up -d --build
+  docker --context $1 compose -f docker-compose.yml -f docker-compose-local.yml logs -f nginx
```

Comment:
```suggestion
  docker --context $1 compose -f docker-compose.yml -f docker-compose-prod.yml up -d --build --force-recreate
```
Fixes the script and also adds the `--force-recreate` flag. This ensures that we do not use cached images when deploying.

Reply:
This is great.